Mathlib/RingTheory/MvPowerSeries/Inverse.lean

```diff
@@ -0,0 +1,293 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Kenny Lau
+-/
+
+import Mathlib.RingTheory.MvPowerSeries.Basic
+import Mathlib.RingTheory.Ideal.LocalRing
+
+
+#align_import ring_theory.power_series.basic from "leanprover-community/mathlib"@"2d5739b61641ee4e7e53eca5688a08f66f2e6a60"
+
+/-!
+# Formal (multivariate) power series - Inverses
+
+This file defines multivariate formal power series
+and develops the basic properties of these objects.
+
+If the constant coefficient of a formal power series is invertible,
+then this formal power series is invertible.
+
+Formal power series over a local ring form a local ring.
+
+-/
```

Comment:
It is probably worth it adding a small implementation note explaining how this inverse is defined, because it is quite counter-intuitive: see the discussion here https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/power.20series/near/346363742

Reply:
It is very clever to exact from the equation phi · (phi inverse) = 1  recurrence relations for the coefficients of (phi inverse) that show how to solve inductively for their coefficients.
I agree some math should be said in the text…